Dallia_Sintique/sistema_notas.py

```diff
@@ -0,0 +1,177 @@
+  status = 0
+  while status == 0:
+    try:
+        if tipo == 'n':
+            nota = float(input(f"Digite a nota {numero_nota} da aluna) (0 a 10): "))
+        else:
+            nota = float(input(f"Digite a nota de participação da aluna) (0 a 10): "))
+        if nota >= 0 and nota <= 10:
+            status = 1
+        else:
+            print("Digite uma nota válida entre 0 e 10")
+    except:
+      print(" Entrada inválida. Digite um número inteiro entre 0 e 10")
+  return nota
+
+def obtem_presenca(num_presenca):
+  contador = 0
+  while contador == 0:
+    presenca = input(f"Digite a presença {num_presenca} da aluna apenas com (True ou False): ").strip()
+    if presenca.lower() in ['true', 'false']:
+      presenca = presenca.lower() == 'true'
+      contador = 1
+    else:
+      print("Entrada inválida, informe a presença apenas com (True ou False)")
+  return presenca
+
+
+def incluir_nova_aluna():
+    print('Iniciando cadastro de nova aluna ')
+    notas = []
+    presencas = []
+
+    nome = obtem_nome(" Informe o nome da aluna para cadastro: ").title()
+    sobrenome = obtem_nome (" Informe o sobrenome da aluna para cadastro: ").title()
+    nome_sobrenome = (nome, sobrenome)
+
+    while True:
+        turma = (input ('Informe a turma da aluna com A / B ou C: ')).strip().upper()
+        if turma == "A" or turma == "B" or  turma == 'C':
+            break
+        else:
+            print('Entrada inválida. Informe uma turma com ( A / B ou C )')
+
+
+    for i in range (1, 4):
+        nota = obtem_nota(i, 'n')
+        notas.append(nota) 
+
+    for i in range (1, 6 ):
+        presenca = obtem_presenca(i)
+        presencas.append(presenca) 
+
+    participacao = obtem_nota(0,'p')
+
+    dataset[nome_sobrenome] = {
+        "Turma": turma ,
+        "Notas": notas,
+        "Presença": presencas ,
+        "Participação": participacao
+    }
+    print("Aluna cadastrada com sucesso ")
+
+
+def consultar_lista_alunas():
+    print(' Segue a lista de alunas cadastradas ')
+    if len(dataset) != 0:
+        for nome in dataset.keys(): 
+            print(nome[0])
+    else:
+        print("Não existem alunas cadastradas")
+
+def consultar_faltas_aluna():
+    try:
+        nome_completo = tuple(obtem_nome('Informe o nome completo da aluna: ').title().split())
+
+        faltas_alunas = dataset[nome_completo]["Presença"]
+
+        numero_faltas = faltas_alunas.count(False)
+        print(f' A aluna {nome_completo[0]} , tem {numero_faltas} faltas')
+    except KeyError:
+        print(" Não localizado. Por favor informe o nome completo de uma aluna: ")
+
+
+def consultar_notas_aluna():
+    try:
+        nome_completo = tuple(obtem_nome('Informe o nome completo da aluna: ').title().split())
+
+        notas_aluna = dataset[nome_completo]['Notas']
+
+        conte_nota = 0
+        for nota in notas_aluna:
+            conte_nota += 1
+            print(f' A nota {conte_nota} da aluna {nome_completo[0]} é: {nota}')
```

Comment:
A função for já enumera sozinho, para recuperar o valor, é só utilizar o enumerate. Assim, não precisamos utilizar o contador e deixamos o código mais limpo.

**Sugestão:**
```suggestion
        for i, nota in enumerate(notas_aluna, start=1):
            print(f'A nota {i} da aluna {nome_completo[0]} é: {nota}')
```